pkg/orchestration/wiring/edge/types.go

```diff
@@ -0,0 +1,46 @@
+package edge
+
+import "github.com/atlassian/voyager"
+
+type InstanceParameters struct {
+	ServiceName voyager.ServiceName `json:"serviceName"`
+	Resource    ResourceParameters  `json:"resource"`
+}
+
+type ResourceParameters struct {
+	Attributes Attributes `json:"attributes"`
+}
+
+type Attributes struct {
+	UpstreamAddress []UpstreamAddress `json:"upstream_address"`
```

Comment:
formatting is off there (Go has strict formatting rules), just run `make fmt` and commit the change